app/controllers/katello/api/v2/repositories_controller.rb

```diff
@@ -139,6 +141,25 @@ def sync
       respond_for_async :resource => task
     end
 
+    api :POST, "/repositories/:id/export", N_("Export a repository")
+    param :id, :identifier, :required => true, :desc => N_("repository ID")
+    param :since, Date, :desc => N_("Optional date of last export (ex: 2010-01-01T12:00:00), useful for exporting deltas. If not specified, a full export will occur."), :required => false
+    def export
+      # ensure we received a parseable date so we can error out before the
+      # tasks are created
+      unless params[:since].blank?
+        begin
+          params[:since].to_time
+        rescue
+          raise HttpErrors::BadRequest, N_("Unable to parse date")
```

Comment:
Perhaps a more user focused message here? The term `parse` reminds me of a programming term compared to something like `Invalid date format provided`. Also, I think this should be `_` instead of `N_`.